oak-blob-cloud/src/test/java/org/apache/jackrabbit/oak/blob/cloud/s3/TestS3Ds.java

```diff
@@ -129,6 +260,43 @@ protected DataStore createDataStore() throws RepositoryException {
         return s3ds;
     }
 
+    protected void doDeleteRecord(DataStore ds, DataIdentifier identifier) throws DataStoreException {
+        ((S3DataStore)ds).deleteRecord(identifier);
+    }
+
+    protected DataRecord doSynchronousAddRecord(DataRecordAccessProvider ds, InputStream in) throws DataStoreException {
+        return ((S3DataStore)ds).addRecord(in, new BlobOptions().setUpload(BlobOptions.UploadType.SYNCHRONOUS));
+    }
+
+    protected File createFile() {
+        long count=0;
```

Comment:
Need for this method? Construct file directly using  
` file = new File("target", "bigfile.txt");
 InputStream testStream = randomStream(0, 256);
 FileIOUtils.copyInputStreamToFile(testStream, file);`